ingress/controllers/nginx/Dockerfile

```diff
@@ -14,10 +14,12 @@
 
 FROM gcr.io/google_containers/nginx-slim:0.8
 
-RUN apt-get update && apt-get install -y \
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
   diffutils \
+  ssl-cert \
   --no-install-recommends \
-  && rm -rf /var/lib/apt/lists/*
+  && rm -rf /var/lib/apt/lists/* \
+  && make-ssl-cert generate-default-snakeoil --force-overwrite
```

Comment:
No particular reason. 
The command  `make-ssl-cert generate-default-snakeoil --force-overwrite` is provided by [make-ssl-cert](http://packages.ubuntu.com/xenial/ssl-cert).
(I'm just trying to avoid reinventing the wheel)